cse/values.yaml

```diff
@@ -5,13 +5,10 @@
 replicaCount: 1
 
 image:
-  repository: virtru/cse
+  repository: docker.io/virtru/cse
```

Comment:
@ehorning just thinking about what could impact Marketplace...would hardcoding docker.io here cause an issue?

Reply:
Ignore, should just take out docker.io since that is default if not specified otherwise.

Reply:
👍 sounds good -- wouldn't cause an issue in marketplace though as that value gets overridden by the value set in the virtru-public values.yaml

Reply:
Ah yeah okay makes sense